test/system/030-run.bats

```diff
@@ -832,6 +832,25 @@ EOF
     is "${lines[0]}" ".*${HOST}.*"
 }
 
+@test "podman run --read-write-tmpfs" {
+    # FIXME once ubuntu gets a newer crun
+    if is_ubuntu; then
+        if [ $(podman_runtime) == "crun" ]; then
+	   skip "The version of crun shipped by ubuntu is too old. Needs crun-1.4.2 or greater"
```

Comment:
I have a `find-obsolete-skips` script that I run periodically. Could you add "FIXME:" to this and the above `skip/Skips` ? That will bring these to my attention when I run the script next time.